pkgs/development/libraries/cairo/default.nix

```diff
@@ -55,6 +55,13 @@ in stdenv.mkDerivation rec {
   ] ++ optionals stdenv.hostPlatform.isDarwin [
```

Comment:
This conditional should be removed to make sure the patches don't bitrot.

Reply:
Done, the check is removed.